components/control/Button/Button.stories.ts

```diff
@@ -0,0 +1,238 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './Button';
+import '../../composition/Grid/Grid';
+import '../../composition/Grid/GridItem';
+
+export default {
+  component: 'diamond-button',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Provides styles for default HTML button elements that can be enhanced by wrapping them in a `diamond-button` component.',
+      },
+    },
+  },
+  argTypes: {
+    size: {
+      control: {
+        type: 'select',
+      },
+      options: ['sm', 'md', 'lg'],
+    },
+    variant: {
+      control: {
+        type: 'select',
+      },
+      options: ['primary', 'secondary', 'text'],
+    },
+    width: {
+      control: {
+        type: 'select',
+      },
+      options: ['full-width', 'full-width-mobile', 'square'],
+    },
+  },
+};
+
+export const Button: StoryObj = {
+  render: (args) => html`
+    <diamond-button size="${args.size}" variant="${args.variant}" width="${args.width}">
+      <button type="button">Button</button>
+    </diamond-wrap>
+  `,
+};
+
+export const AnchorButton: StoryObj = {
+  render: () => html`
+    <diamond-button>
+      <a href="#" onclick="return false;">Anchor</a>
+    </diamond-button>
+  `,
+};
+
+export const PrimaryButton: StoryObj = {
+  render: () => html`
+    <diamond-grid wrap="wrap">
+      <diamond-grid-item>
+        <diamond-button variant="primary">
+          <button>Primary</button>
+        </diamond-button>
+      </diamond-grid-item>
+      <diamond-grid-item>
+        <diamond-button variant="primary">
+          <button disabled>Disabled</button>
+        </diamond-button>
+      </diamond-grid-item>
+    </diamond-grid>
+  `,
+};
+
+export const SecondaryButton: StoryObj = {
+  render: () => html`
+    <diamond-grid wrap="wrap">
+      <diamond-grid-item>
+        <diamond-button variant="secondary">
+          <button>Secondary</button>
+        </diamond-button>
+      </diamond-grid-item>
+      <diamond-grid-item>
+        <diamond-button variant="secondary">
+          <button disabled>Disabled</button>
+        </diamond-button>
+      </diamond-grid-item>
+    </diamond-grid>
+  `,
+};
+
+export const TextButton: StoryObj = {
+  render: () => html`
+    <diamond-grid wrap="wrap">
+      <diamond-grid-item>
+        <diamond-button variant="text">
+          <button>Text</button>
+        </diamond-button>
+      </diamond-grid-item>
+      <diamond-grid-item>
+        <diamond-button variant="text">
+          <button disabled>Disabled</button>
+        </diamond-button>
+      </diamond-grid-item>
+    </diamond-grid>
+  `,
+};
+
+export const IconOnlyButton: StoryObj = {
+  render: () => html`
+    <diamond-grid wrap="wrap">
+      <diamond-grid-item>
+        <diamond-button width="square">
```

Comment:
Should we be explicit and say width icon?

Reply:
I don't think we need to tie it to an icon. Width icon could do anything, square is a shape everyone knows